docs/pt/docs/tutorial/encoder.md

```diff
@@ -0,0 +1,42 @@
+# Codificador Compatível com JSON
+
+Existem alguns casos em que você pode precisar converter um tipo de dados (como um modelo Pydantic) para algo compatível com JSON (como um `dict`, `list`, etc).
```

Comment:
Are we using "voce" in the docs? 🤔

Reply:
Hi Kludex, I didn't understand your comment. Are you referring to a typo, or page formality?

If it is about the formality, just let me know that I'll fix it. However,
answering your question, others pt translated pages in the docs also uses "você", so I thought it wouldn't be a problem.

Anyway, let me know how you would like it to be. Thanks!

Reply:
Ok then.